homeassistant/components/ruckus_unleashed/manifest.json

```diff
@@ -1,10 +1,11 @@
 {
   "domain": "ruckus_unleashed",
   "name": "Ruckus Unleashed",
-  "codeowners": ["@gabe565"],
+  "codeowners": ["@gabe565", "@lanrat"],
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/ruckus_unleashed",
+  "integration_type": "hub",
   "iot_class": "local_polling",
-  "loggers": ["pexpect", "pyruckus"],
-  "requirements": ["pyruckus==0.16"]
+  "loggers": ["aioruckus", "xmltodict"],
+  "requirements": ["aioruckus==0.31", "xmltodict==0.13.0"]
```

Comment:
Why do we require xmltodict? If aioruckus requires xmltodict, aioruckus should include it in its own requirements.